docs/installing-solidity.rst

```diff
@@ -174,11 +174,12 @@ Install it using ``brew``:
     # eg. Install 0.4.8
     brew install https://raw.githubusercontent.com/ethereum/homebrew-ethereum/77cce03da9f289e5a3ffe579840d3c5dc0a62717/solidity.rb
 
-Gentoo Linux also provides a solidity package that can be installed using ``emerge``:
+Gentoo Linux has an overlay at https://overlays.gentoo.org/#ethereum that contains a solidity package.
```

Comment:
```suggestion
Gentoo Linux has an overlay at `https://overlays.gentoo.org/#ethereum <https://overlays.gentoo.org/#ethereum>`- that contains a solidity package.
```

To make the link clickable

Reply:
Nice, thanks.
I also turned the clickable piece into regular text to ease reading.